azure/pipelines/dotnet/ci.yml

```diff
@@ -186,38 +165,34 @@ steps:
     inputs:
       command: 'publish'
       projects: ${{ parameters.Solution }}
-      # Version .NET SDK 7.0.200 does not support output parameter on solution builds.
-      # See https://learn.microsoft.com/en-us/dotnet/core/compatibility/sdk/7.0/solution-level-output-no-longer-valid.
-      # Starting from SDK 7.0.201, this has been degraded to a warning for other dotnet commands.
-      # As we really want to perform solution builds/publishes and as long we have build agents that runs on 7.0.200,
-      # we use the workaround to specify the 'PublishDir' property.
       arguments: '--configuration ${{ parameters.BuildConfiguration }} ${{ parameters.PostBuildDotnetArgs }} --property PublishDir=$(Build.ArtifactStagingDirectory)/packages'
       publishWebProjects: false
       modifyOutputPath: false
       zipAfterPublish: false
 
 - ${{ if ne(parameters.NoTests, 'true') }}:
   - ${{ each step in parameters.BeforeTestsSteps }}:
-    - ${{ step }} 
-  - ${{ if and( eq(parameters.NoTests, false), ne(parameters.WithPlatinaTestInit, false) ) }}:
-    - task: PowerShell@2
-      displayName: 'Platina End Test Initialization'
-      inputs:
-        targetType: 'inline'
-        script: |
-          Get-ChildItem -Filter ${{ parameters.Solution }} | Foreach-Object {
-            dotnet msbuild /target:EndPlatinaTestInit "$($_.FullName)"
-          }
-          # Ignore msbuild errors because targets might not be present in projects.
-          exit 0;
+    - ${{ step }}
+  
+  # Sets test coverage default when not provided.
+  # Defaults depends whether MTP is configured with .NET 10 or not.
+  # Use '--coverage' for .NET 10 and higher, '--collect "Code Coverage"' for lower versions.
+  # TBD: For now, assume that MTP runner is used when NET 10 and higher?
```

Comment:
That could be the default, but we should still allow the parameter to be set by the calling ci. I don't think this is supported anymore.